keps/sig-cluster-lifecycle/kubeadm/0015-kubeadm-join-control-plane.md

```diff
@@ -417,6 +409,12 @@ one control plane instances, but with a new sub-step to be executed on secondary
 - [PR #58261](https://github.com/kubernetes/kubernetes/pull/58261) with the showcase implementation of the first release of this KEP
 - v1.12 first implementation of `kubeadm join --control plane`
 - v1.13 support for local/stacked etcd
+- v1.14 implementation of automatic certificates copy
+  (see KEP [Certificates copy for join --control-plane](20190122-Certificates-copy-for-kubeadm-join--control-plane.md)).
+- v1.15 (planned) graduate to beta
```

Comment:
Strictly speaking this has not happened yet, so it should not be in the implementation history. I'd rather have it after 1.15 release (or rather what we have done there).